### PR TITLE
Adding ShipmentMode fix in the changelog

### PR DESCRIPTION
**Pull request by delias-silva**
## Description
Adding ShipmentMode fix in the changelog

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/resources/changelog/current-year.es.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
 
 ## Febrero 2021
 
+### February 16th
+
+> CHANGELOG
+>
+> Se corrigió la serialización de la enumeración ShipmentMode en el SDK .NET
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.es.md:13]:
```suggestion
> Cambios en la serialización de la enumeración ShipmentMode
```

File: guides/resources/changelog/current-year.es.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
...
+>
+> NEWS: NOVEDADES
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.es.md:15]:
```suggestion
> FIX: FIX
```

File: guides/resources/changelog/current-year.es.md [hunk 3]
```diff
@@ -6,6 +6,20 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
...
+>
+> PRODUCT: SDK .NET
+
+Arreglamos la serialización de la enumeración ShipmentMode para generar valores aceptados por la API de Preference
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.es.md:19]:
```suggestion
Arreglamos la serialización de la enumeración del `ShipmentMode` en la SDK .Net C# para generar valores aceptados por la API de Preferencias.
```

**Comment by pamalv7** [comment]
@delias-silva Dejé una propuesta de iteración posible. ¿Lo pueden revisar @mminestrelli y @abderossi?

**Review by abderossi — APPROVED** [review]

File: guides/resources/changelog/current-year.es.md [hunk 4]
```diff
@@ -6,6 +6,20 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
...
+>
+> Cambios en la serialización de la enumeración ShipmentMode
+>
+> FIX: FIX
+>
+> PRODUCT: SDK .NET
+
+Arreglamos la serialización de la enumeración del `ShipmentMode` en la SDK .Net C# para generar valores aceptados por la API de Preferencias.
+
+[Go to SDKs documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/sdks)
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.es.md:21]:
```suggestion
[Ir a documentación de las SDKs](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/sdks)
```

File: guides/resources/changelog/current-year.en.md [hunk 5]
```diff
@@ -6,6 +6,20 @@ Find out everything about the new versions and updates of Mercado Pago integrati
 
 ## February 2021
 
+### February 16th
+
+> CHANGELOG
+>
+> Fixed serialization of the ShipmentMode enum in the .NET SDK
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.en.md:13]:
```suggestion
> Changes to ShipmentMode Enum Serialization
```

File: guides/resources/changelog/current-year.en.md [hunk 6]
```diff
@@ -6,6 +6,20 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+>
+> NEWS: NEWS
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.en.md:15]:
```suggestion
> FIX: FIX
```

File: guides/resources/changelog/current-year.en.md [hunk 7]
```diff
@@ -6,6 +6,20 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+>
+> PRODUCT: SDK .NET
+
+We have fixed ShipmentMode enum serialization to generate values accepted by the Preference API
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.en.md:19]:
```suggestion
We have fixed the `ShipmentMode` enum serialization in the .Net C # SDK to generate values accepted by the Preferences API.
```

File: guides/resources/changelog/current-year.pt.md [hunk 8]
```diff
@@ -6,6 +6,20 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
 
 ## Fevereiro 2021
 
+### February 16th
+
+> CHANGELOG
+>
+> Foi corrigida a serialização do enum ShipmentMode no SDK .NET
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:13]:
```suggestion
> Mudanças na serialização do enum ShipmentMode
```

File: guides/resources/changelog/current-year.pt.md [hunk 9]
```diff
@@ -6,6 +6,20 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
...
+>
+> NEWS: NOVIDADES
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:15]:
```suggestion
> FIX: FIX
```

File: guides/resources/changelog/current-year.pt.md [hunk 10]
```diff
@@ -6,6 +6,20 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
...
+>
+> PRODUCT: SDK .NET
+
+Corrigimos a serialização do enum ShipmentMode para gerar valores aceitos pela API de Preference
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:19]:
```suggestion
Corrigimos a serialização do enum `ShipmentMode` no .Net C # SDK para gerar valores aceitos pela API de preferências.
```

File: guides/resources/changelog/current-year.pt.md [hunk 11]
```diff
@@ -6,6 +6,20 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
...
+
+[Go to SDKs documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/sdks)
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:21]:
```suggestion
[Ir para a documentação das SDKs](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/sdks)
```

File: guides/resources/changelog/last-year.en.md [hunk 12]
```diff
@@ -6,6 +6,20 @@ Find out everything about the new versions and updates of Mercado Pago integrati
 
 ## February 2021
 
+### February 16th
```
Comment (pamalv7) [inline guides/resources/changelog/last-year.en.md:9]: Esto es incorrecto. Solo aplica cambio para `current-year`

Comment (delias-silva) [reply guides/resources/changelog/last-year.en.md]: Quite los cambios en `last-year`

**Pull request merged**